src/encord_active/app/cli/utils.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+from pathlib import Path
+from typing import Any, Callable, TypeVar
+
+R = TypeVar("R")
+funcs = {}
+
+
+def bypass_streamlit_question(fn: Callable[..., R]) -> Callable[..., R]:
+    funcs[fn.__name__] = fn
```

Comment:
[curious]🤔 

This seems unused, is that needed?

Reply:
Should be imported by `.../cli/main.py` right?